### PR TITLE
chore: quick fix on notation policy command print out

### PR DESCRIPTION
**Pull request by Two-Hearts**

### CODE REVIEW

**Comment by priteshbandi** [comment]
Can you please share before and after fix cli output sample ?

**Comment by Two-Hearts** [comment]
> Can you please share before and after fix cli output sample ?

@priteshbandi the change is adding a newline character. For example, if the import trustpolicy is invalid, then the output message is:
`Warning: existing trust policy configuration file will be overwrittenError: failed to validate trust policy:...`

File: cmd/notation/policy/import.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func runImport(command *cobra.Command, opts importOpts) error {
 			}
 		}
 	} else {
-		fmt.Fprintf(os.Stderr, "Warning: existing trust policy configuration file will be overwritten")
+		fmt.Fprintf(os.Stderr, "Warning: existing trust policy configuration file will be overwritten\n")
```
Comment (shizhMSFT) [inline cmd/notation/policy/import.go:68]: Can we use `Fprintln` instead?

Comment (Two-Hearts) [reply cmd/notation/policy/import.go:68]: Yeah, that will also work. Updating the code.

**Review by shizhMSFT — DISMISSED** [review]
LGTM

**Review by priteshbandi — DISMISSED** [review]
LGTM

**Comment by priteshbandi** [comment]
@Two-Hearts looks like `5e1c87ef2829276059e9b131193cd03ff6c70119` commit is not signed

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Comment by Two-Hearts** [comment]
> @Two-Hearts looks like `5e1c87ef2829276059e9b131193cd03ff6c70119` commit is not signed

oops, my bad. Signed the commits.

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**